core/blockly.js

```diff
@@ -90,6 +90,13 @@ Blockly.clipboardTypeCounts_ = null;
  */
 Blockly.cache3dSupported_ = null;
 
+/**
+ * Container element to render the WidgetDiv, DropDownDiv and Tooltip.
```

Comment:
This should be lower case.  It's not a constant.

Reply:
True.